api/src/test/java/org/openmrs/api/handler/PatientDataUnvoidHandlerTest.java

```diff
@@ -144,4 +149,39 @@ public void handle_shouldNotUnvoidTheOrdersAndEncountersThatNeverGotVoidedWithTh
 		Assert.assertNotNull(testOrder.getVoidReason());
 		
 	}
+	
+	/**
+	 * @see PatientDataUnvoidHandler#handle(Patient,User,Date,String)
+	 */
+	@Test
+	@Verifies(value = "should unvoid the members associated with the patient", method = "handle(Patient,"
+			+ "User,Date,String)")
```

Comment:
@vshl you added tests the old style. please use
https://wiki.openmrs.org/display/docs/Generate+Test+Case+Plugin
to adhere to the new style